packages/calcite-components/src/components/accordion/accordion.scss

```diff
@@ -1,41 +1,77 @@
+/**
+ * CSS Custom Properties
+ *
+ * @prop --calcite-accordion-space: defines space around and between accordion item elements. Other spacing is calculated at a +4.
+ * @prop --calcite-accordion-border-color: The border around an accordion.
+ * @prop --calcite-accordion-background: The background color of the accordion and accordion items.
+ * @prop --calcite-accordion-header-text-color: The header text color for accordion items.
```

Comment:
These `heading` and `description` props are set on the Accordion Items themselves, do we need these at this level? If so, these don't seem to match the Accordion Item variable names.